### ISSUE TITLE
wrong data saved to file

### ISSUE THREAD

**Issue by finhj** [olivierkes/manuskript#499]
Somehow I ended up with two instances of Manuskript running - I think after right-clicking and opening a scene in a new tab, but I can't reproduce it. I didn't notice until I was finished working.

I'd saved the file a number of times, saved it one last time and closed it, and then I noticed the other window. I never did any work from that window, and I didn't save from it, but it was that data that was saved, not the data from the instance I had been working in.

So, to recap - one file, two Manuskript windows/instances. Saving from the the first window caused data from the second window to be saved to the file.

I was using Manuskript 0.8.0 in Manjaro 4.19.20.

**Comment by gedakc** [comment]
By default manuskript automatically saves changes to projects.  At present having multiple concurrent instances of Manuskript running is not supported.  The consequences are unpredictable as any instance can write data to the project file(s) at any time.

Marking this bug as a duplicate of [Issue 315 - File Load Failure - Corrupted? Multiple Concurrent Manuskript Instances not supported](https://github.com/olivierkes/manuskript/issues/315).
